src/cmd/config.go

```diff
@@ -15,6 +15,7 @@ var setupFlag bool
 var configCommand = &cobra.Command{
 	Use:   "config",
 	Short: "Displays or updates your Git Town configuration",
+	Long:  `Displays or updates your Git Town configuration`,
```

Comment:
I don't think this is needed as the short form is used if the long is not present